setup.py

```diff
@@ -47,6 +52,11 @@ def generate_long_description() -> str:
     url="https://github.com/ericmjl/pyjanitor",
     packages=["janitor"],
     install_requires=requirements(),
+    extras_require={
+            'all': extra_all,
+            'biology': extra_biology,
+            'chemistry': extra_chemistry,
+            'engineering': extra_engineering}
```

Comment:
Missing comma is causing build to fail.

```suggestion
            'engineering': extra_engineering},
```

Reply:
to be honest i dont know, I will need to check both

Reply:
No worries.

If you want, you can use the "commit suggestion" functionality to accept the suggestion directly without needing to touch your text editor :smile:.